examples/gitlab/redis.yml

```diff
@@ -0,0 +1,13 @@
+name: redis
+containers:
+  - image: redis:3.2.4
+    volumeMounts:
+    - name: db
+      mountPath: /var/lib/redis
+services:
+  - name: redis
+    ports:
+    - port: 6379
+volumes:
+- name: db
+  emptyDir: {}
```

Comment:
This will lose data if the container is moved to another node.
`emptyDir` dir shouldn't be used here.